src/main/test/latmap/LongSolverTest.scala

```diff
@@ -41,18 +33,52 @@ class LongSolverTest extends FunSuite {
       val t = latVariable(SULattice)
       val z = latVariable(SULattice)
 
+      // Indexes
+      AddrOf(a,b).addIndex(a,b)
+      Copy(a,b).addIndex(a,b)
+      Copy(a,b).addIndex(a)
+      Store(a,b,k).addIndex(a,b,k)
+      Store(a,b,k).addIndex(b)
+      Store(a,b,k).addIndex(k)
+      Load(a,b,k).addIndex(a,b,k)
+      Load(a,b,k).addIndex(a)
+      Load(a,b,k).addIndex(k)
+      CFG(a,b).addIndex(a,b)
+      CFG(a,b).addIndex(a)
+      CFG(a,b).addIndex(b)
+      FIStore(a,b,k).addIndex(a)
+      FIStore(a,b,k).addIndex(b)
+      FILoad(a,b,k).addIndex(b)
+
+      Pt(a,b).addIndex(a,b)
+      Pt(a,b).addIndex(a)
+
+      val SU = relation(2, SULattice)
```

Comment:
Is it necessary to move these relation declarations down here? They make more sense earlier (so that all the relation declarations are together, before the indexes start).

Reply:
I tried to keep this test case code as close to the `SUopt.flix` code as possible - so that you can see what each line of flix code mapped to. If that's not really an important trait to maintain, I can go ahead and reorder all the relation declarations together, followed by variable declarations, followed by indexes.